build.sbt

```diff
@@ -36,6 +36,46 @@ lazy val commonSettings = Seq(
 
 publish / skip := true
 
+lazy val scalaVersionAxis = settingKey[Option[String]]("Project scala version")
+
+lazy val circeAkkaSerializer = (projectMatrix in file("circe-akka-serializer"))
+  .settings(name := "circe-akka-serializer")
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+        akkaTyped % Provided,
+        akkaTestKit % Test,
+        akkaStream % Provided,
+        circeCore,
+        circeParser,
+        circeGeneric % Test,
```

Comment:
So, should we force the inclusion of these libraries? Circe doesn't give any guarantees, it would be much safer just to make them normal dependencies.

Reply:
Yeah, let's do it this way.... at least the end-user will see an sbt eviction warning if they try using a different version of `circe-generic`